CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# 🤝 Contributing to octasol
+
+Welcome to Octasol! We are glad about your interest in contributing. Please take a look at this document to support an easy and effective contribution process for everyone.
+
+Following these guidelines shows respect for our maintainer's time, and in turn, they will respond to your issues and evaluate your contributions.
+
+> [!NOTE] 
+> Please submit one Pull Request per issue! 
+
+<p>&nbsp;</p>
+
+## 👀 Table of Contents
+1. [Setting Up Local Development Environment](#setting-up-local-development-environment)
+1. [Pull Requests Process](#pull-requests-process)
+1. [Code Guidelines](#code-guidelines)
+1. [Test your Code](#test-your-code)
+1. [How to Report Bugs](#how-to-report-bugs)
+1. [Your Help is needed](#your-help-is-needed)
+
+<p>&nbsp;</p>
+
+## Setting Up Local Development Environment
+
+
+1. Fork the Repository
+1. Clone the Repository
+
+    ```
+    git clone https://github.com/<yourusername>/octasol.git <octasol>
```

Comment:
please remove < and >

Reply:
Thanks for the feedback. I pushed a new commit considering your feedback